src/operator/quantization/quantized_conv.cc

```diff
@@ -119,7 +138,7 @@ and max thresholds representing the threholds for quantizing the float32 output
     return param.no_bias? 6 : 9;
   })
 .set_num_outputs(3)
-.set_attr_parser(ParamParser<ConvolutionParam>)
+.set_attr_parser(ConvolutionParamParser)
```

Comment:
Why cannot the previous param parser be used?

Reply:
After syncing with @zheng-da, the previous ParamParser cannot pass the param **dilate** correctly.